### PR TITLE
Remove defaults for PageTitle and TestName

### PR DESCRIPTION
**Pull request by jfer-siteimprove**
Resolves II-8339

### CODE REVIEW

**Comment by jfer-siteimprove** [comment]
!pr extract

**Review by Jym77 — DISMISSED** [review]

**Comment by Jym77** [comment]
@jfer-siteimprove I looked into it, `Option` don't really make sense here indeed. I removed some parameters that have become useless in various iterations.

**Comment by Jym77** [comment]
!pr extract

**Review by rcj-siteimprove — DISMISSED** [review]

File: packages/alfa-test-utils/src/report/sip.ts [hunk 1]
```diff
@@ -222,18 +220,25 @@ export namespace SIP {
...
+      const name = options.testName ?? Defaults.Name;
       const TestName =
         // If the name is a string, using, otherwise call the function on the
         // gitInfo, defaulting to the error if any.
         typeof name === "string"
           ? name
-          : gitInfo.map(name).getOrElse(() => gitInfo.getErrOr(defaultName));
+          : name !== undefined
+          ? gitInfo.map(name).getOrElse(() => gitInfo.getErrOr(Defaults.Name))
+          : gitInfo.getErrOr(Defaults.Name);
```
Comment (Jym77) [inline packages/alfa-test-utils/src/report/sip.ts:241]:
```suggestion
          : Defaults.Name;
```

**Review by Jym77 — CHANGES_REQUESTED** [review]
Small clean up.

**Review by Jym77 — APPROVED** [review]

**Pull request merged**